browser/components/smartwindow/content/chat.mjs

```diff
@@ -641,13 +673,21 @@ Examples:
 Always provide a helpful response first, then include the search suggestion when appropriate.
 
 # Insights and Personalization Rules
-${buildInsightsSystemPrompt()}
+${buildInsightsSystemPrompt()}`;
+
+    if (includeTitleGeneration) {
+      systemPrompt += `\n\n# Title Generation Rules
+At the start of your response, you must create a concise title for the conversation based on the user's message.
+The title should be less than 6 words and should reflect the main topic or intent of the user's message.
+Do not end with punctuation (no period, question mark, etc.). Do not generate questions as titles.
+
+Format the title as follows: §title: title§`;
```

Comment:
other parts of this system prompt use [[search: value]] syntax. do you see § works better? we have run into issues with qwen hallucinating `[[search: query [placeholder] stuff]]` maybe biased by the square bracket syntax